tools/reviewer/job/tasks/ReviewerMetadataUpdaterTask.java

```diff
@@ -153,5 +153,178 @@ public void run() {
   public Boolean shouldRun() {
     return !lock.isLocked();
   }
-}
 
+  public ReviewerConfig printStartupOsReviewerConfig() throws IOException {
```

Comment:
I think the main aim of this method is getting `ReviewerConfig` message. Please rename the method. (`print...` -> `get...`)

Reply:
You're right. I thought about it, but forgot to do it.

Reply:
Fixed. See comment below.